layers/+web-services/llm-client/packages.el

```diff
@@ -44,18 +44,18 @@
     :defer t
     :ensure t
     :init
-    ;; evilify gptel-context-buffer-mode-map
-    (require 'gptel-context)
-    (evil-set-initial-state 'gptel-context-buffer-mode 'evilified)
-    (evilified-state-evilify-map gptel-context-buffer-mode-map
-      :mode gptel-context-buffer-mode
-      :bindings
-      "C-c C-c" #'gptel-context-confirm
-      "C-c C-k" #'gptel-context-quit
-      "RET"     #'gptel-context-visit
-      "n"       #'gptel-context-next
-      "p"       #'gptel-context-previous
-      "d"       #'gptel-context-flag-deletion)
+    (with-eval-after-load 'gptel-context
+      ;; evilify gptel-context-buffer-mode-map
+      (evil-set-initial-state 'gptel-context-buffer-mode 'evilified)
```

Comment:
I think `evil-set-initial-state` works fine before the mode has been defined.  But, FWIW, `evilified-state-evilify-map` also already sets up the initial state so I think this line can just be deleted.